src/unity/python/turicreate/toolkits/drawing_classifier/drawing_classifier.py

```diff
@@ -197,6 +197,18 @@ def create(input_dataset, target, feature=None, validation_set='auto',
         raise TypeError("Unrecognized type for 'validation_set'."
             + validation_set_corrective_string)
 
+
+    original_dataset_size = len(dataset)
+    dataset = dataset.dropna(columns=feature)
```

Comment:
Won't this mutate the original dataset? I think we should not change the user data.

Reply:
I don't see how. `dropna` returns a new SFrame, doesn't change things in place.

Reply:
It doesn't mutate the data